release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -517,11 +517,21 @@ For more information, see _Configuring Capacity Reservation by using machine set
 [id="ocp-4-17-nodes_{context}"]
 === Nodes
 
+[id="ocp-4-17-must-gather-new-flags_{context}"]
+=== New flags added for must-gather command
+
+{product-title} release {product-version} adds two new flags for use with the `oc adm must-gather` command to limit the timespan of the information gathered. The new flags and their uses are listed here. Only one of the following flags can be used at a time. Plugins are encouraged but not required to support these flags.
+
+* `--since`: Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs.
```

Comment:
```suggestion
* `--since`: Only return logs newer than a relative duration, such as 5s, 2m, or 3h. Defaults to all logs.
```
IBM Style word usage says to avoid using _like_ to mean _such as_.